docs/software/apple/usage.mdx

```diff
@@ -3,6 +3,71 @@ id: usage
 title: Apple Application Usage
 sidebar_label: Usage
 ---
+## Intro
+
+There is a Meshtastic app in the Apple App Store for iPhones, iPads and Macs (on their respective OSs: iOS, iPadOS, and macOS) with slightly different behaviour on each platform. On iOS and iPadOS, V16.2 or greater is required; macOS requires Sonoma.
+
```

Comment:
Lots of repetition here. Perhaps something like:
The Meshtastic App is available for download in the App Store for iPhone, iPad and Mac. Meshtastic requires iOS/iPadOS 16.2 or macOS Sonoma.